scripts/docker/README.md

```diff
@@ -0,0 +1,96 @@
+# sdk-nrf-toolchain docker image
+
+## Content
+
+The image is based on ubuntu:22.04 docker image with additional tools:
+
+* git
+* nrfutil with following commands:
+    * toolchain-manager
+    * device
+* ncs toolchain-bundle
+* JLink (installation requires accepting SEGGER License)
+
+## Building image
+
+To build docker image you have to provide `VERSION` argument containing ID of Linux toolchain bundle you want to
+install.
+You can use [print_toolchain_checksum.sh](../print_toolchain_checksum.sh) to get ID proper for your sdk-nrf revision.
+
+```shell
+TOOLCHAIN_ID=$(../print_toolchain_checksum.sh)
+docker build --build-arg VERSION=$TOOLCHAIN_ID .
+```
+
+## Accepting JLink License
+
+To install and use JLink software you have to accept [JLink License Agreement](./jlink/license.txt).
+
+```
+By accessing and using Software and Materials provided by SEGGER as free download, you acknowledge and agree to the following Terms of Use. If you do not agree to these Terms, do not download or use any Software or Material.
```

Comment:
This is same as license. Cannot we just link to license file here or ideally leave whole license out from our repo?

Reply:
License is linked, but I wanted to print it here too. 
Ideally, I would like to remove license file from sdk-nrf repository, but I could not find any link to Terms of Use.